api/swagger.yml

```diff
@@ -439,6 +439,9 @@ components:
           type: object
           additionalProperties:
             type: string
+        strategy:
+          description: changes from "ours"(dest) or "theirs"(source) are automatically overridden in case of a conflict
```

Comment:
1. The terms `ours` and `theirs` were selected to align with a similar `git` [feature](https://git-scm.com/docs/merge-strategies), so IMO we should keep these. WDYT?
2. How about ```In case of a merge conflict, this option will make the merge process to automaticall favor changes from the dest branch (ours) or from the source branch(theirs). In case no selection is made, the merge will fail in case of a conflict``` - @talSofer - WDYT?

Reply:
Changed the description string to the one above. Let me know what you think

Reply:
I also think we should stick to git terminology, and this description is better :)

Reply:
I like the new description, thanks!

I still think that "ours" and "theirs" is not meaningful in the context of lakefs (since we don't have the concept of a local environment) and it adds another mental hop users should do. for example, if the source is staging branch and dest is production branch. as a user, I need to remember that ours is production and theirs is staging.. IMO this is not clear :)  

Users thought it is clear [here](https://github.com/treeverse/lakeFS/issues/2739) so maybe it's only me. anyways, I think it worth asking for more opinions.